src/loop/typer.ml

```diff
@@ -703,16 +712,31 @@ let unhandled_ast : (T.env * Dolmen_std.Term.t T.fragment) Report.Error.t =
           print_fragment (env, fragment))
     ~name:"Unhandled AST fragment" ()
 
+let bad_farray_arity =
+  Report.Error.mk ~code ~mnemonic:"bad-farray-arity"
+    ~message:(fun fmt _ ->
+        Format.fprintf fmt "Functional array types in Alt-Ergo expect either one or two type \
+        parameters.")
+    ~hints:[text_hint]
```

Comment:
it doesn't look like this hint is actually used, so it can probably go away (and the parameter of this error turned into `unit`)

Reply:
Done.